src/wagtail_live/adapters/slack/receiver.py

```diff
@@ -88,16 +88,14 @@ def dispatch_event(self, event):
         """See base class."""
 
         message = event["event"]
-        if "subtype" in message and message["subtype"] == "message_changed":
-            self.change_message(message=message)
+        if subtype := message.get("subtype"):
```

Comment:
[The walrus operator (`:=`) was introduced in Python 3.8](https://docs.python.org/3/whatsnew/3.8.html#assignment-expressions) and is still quite new. Not everyone using Wagtail is going to be on Python 3.8 (speaking from experience 😇), so I'd exercise caution in using this syntax.

**Suggestion:** could we change this to be compatible with older Python versions like 3.7?

Reply:
Alright!